app/assets/stylesheets/apply.scss

```diff
@@ -55,6 +55,26 @@
   background-size:cover;
 }
 
+.gear-img {
+  background: image-url('apply/gears.svg');
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Prefer double-quoted strings